_layouts/pattern.html

```diff
@@ -38,23 +45,23 @@
   {% endif %}
 </div>
 
-<div class="grid-x grid-padding-x">
+<div class="ie-content grid-x grid-padding-x">
   <div class="large-8 large-offset-2 small-12 cell">
-    <h2 class="no-margin">{{ page.title }}</h2>
+    <h2 class="ie-title no-margin">{{ page.title }}</h2>
     <p><a href="/category/{{ page.category | slugify }}" class="border-{{ page.category | slugify }}">{{ page.category }}</a></p>
 
     {{ content }}
 
   </div>
 </div>
 
-<div class="grid-x grid-padding-x">
+<div class="ie-content grid-x grid-padding-x">
 
     {% if page.advantages %}
     <div class="large-4 large-offset-2 small-12 cell">
-      <h3>Advantages</h3>
+      <h3 class="ie-h3">Advantages</h3>
 
-      <ul>
+      <ul class="ie-list">
```

Comment:
Great idea to prefix classes for IE with `ie-`. Really useful to be able to scan the HTML and quickly now which classes are generally used and which ones are IE only. Hopefully this will make it easier to maintain